### PR TITLE
fix: inconsistent uppercase issue resolved

### LINKED ISSUE #11115: Issue/Suggestion: "Inconsistent Uppercase Usage"
Issue/Suggestion: "Inconsistent Uppercase Usage"

### Issue Summary

 In some parts, the words are not in uppercase where they should be.
 I suggest that avatar should be Avatar. and Connect your calendar should be Connect your  Calendar.

### Evidence

Screenshot (174)
Screenshot (175)

### PR DESCRIPTION
**Pull request by DevFaizann**
Fixes #11115 

image
image

## How should this be tested?

Visit the getting-started page.
In step 2, you will see the issue resolved.

Complete all the steps and go to the profile settings page
Click on upload avatar, and you will see the issue resolved.

## Mandatory Tasks

- [ X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by hariombalhara — CHANGES_REQUESTED** [review]
cc @PeerRich to approve the copy changes. Let's not merge without his approval.

**Comment by PeerRich** [comment]
i think we should keep normal english: "Connect your calendar"

**Comment by DevFaizann** [comment]
> i think we should keep normal english: "Connect your calendar"

@PeerRich  thank you for your feedback and consideration. i appreciate your perspective on this matter.

**Pull request closed without merging**
